uiowa/uiowa
# Allow for minimal text in CTA

Resolves #2710 




# How to test

- Sync a site, make sure update hook / config updates fire without issue
- Check existing CTAs on the site, and see that their existing summaries are still present and set to `plain_text`
- Create a new CTA, check that it defaults to `plain_text`
- Try saving a CTA with `minimal` text format, check that it saves correctly.
   - Check that when editing the newly created block, prior to saving the layout, it retains `minimal` text format
   - Check that the CTA content doesn't receive extra margin spacing (the `<p>` when using `minimal` should be styled to `margin: 0`



Some preliminary test CTAs are available on the sandbox site at https://sandbox.prod.drupal.uiowa.edu/calls-action
